### ISSUE TITLE
Double ulimit wrapper

### ISSUE THREAD

**Issue by Gbury** [sneeuwballen/benchpress#39]
Currently it seems that provers run by benchpress are wrapped **twice** in a ulimit wrapper, which is at the very least superfluous, and I suspect might be one reason why I continue to have zombie processes when running benchmarks.

Additionally, the two wrappers do not enforce the same limits, which may or may not make things even more confusing.

For the record here is the command run for a timeout of 600s and 2G memory limit, as per benchpress logs:
```
cmd "ulimit -t 600 -m 2000000 -Sv 16000000; ulimit -t $(( 1 + 600)) -v $(( 1200 * 2000 )); <prover_cmd>
```

**Comment by Gbury** [comment]
Additionally, after a more careful reading, the first/outermost ulimit seems to not use the correct options (using `-m` instead of `-v` for the given memory limit and using a constant for `-v`)

**Comment by Gbury** [comment]
Solved by #41
